apps-rendering/src/components/Standfirst/index.tsx

```diff
@@ -1,219 +1,77 @@
-	margin-bottom: 0;
-	padding-bottom: ${remSpace[3]};
-
-	p {
-		margin: 0;
-		padding: 0.75rem 0;
-
-		${darkModeCss`color: ${neutral[93]};`}
-	}
-
-	ul {
-		margin-bottom: 0;
-	}
-`;
-
-const deadblogStyles = (format: ArticleFormat): SerializedStyles => {
-	const colour = text.standfirstLink(format);
-
-	return css`
-		${headline.xxxsmall({ fontWeight: 'bold' })};
-
-		a {
-			text-decoration: none;
-		}
-
-		a:link {
-			color: ${colour};
-			border-bottom: 1px solid ${neutral[86]};
-		}
-
-		a:hover {
-			color: ${colour};
-			border-bottom: 1px solid ${colour};
-		}
-
-		${darkModeCss`
-			a:link, a:hover {
-				color: ${text.standfirstLinkDark(format)};
-				border-bottom-color: ${border.standfirstLinkDark(format)};
-			}
-		`}
-	`;
-};
-
-const media = (format: ArticleFormat): SerializedStyles => css`
-	color: ${text.standfirst(format)};
-	p,
-	ul,
-	li {
-		${headline.xxxsmall({ fontWeight: 'bold' })}
-	}
-`;
-
-const advertisementFeature = css`
-	${textSans.medium()}
-`;
-
-const getStyles = (item: Item): SerializedStyles => {
+const Standfirst: React.FC<Props> = ({ item }) => {
 	const format = getFormat(item);
-	if (item.display === ArticleDisplay.Immersive) {
-		return item.theme === ArticleSpecial.Labs
-			? css(styles(format), immersiveLabs)
-			: css(styles(format), immersive);
-	}
 
-	if (item.theme === ArticleSpecial.Labs) {
-		return css(styles(format), advertisementFeature);
+	if (format.display === ArticleDisplay.Immersive) {
+		return format.theme === ArticleSpecial.Labs ? (
+			<ImmersiveLabsStandfirst item={item} />
+		) : (
+			<ImmersiveStandfirst item={item} />
+		);
 	}
-
-	switch (item.design) {
+	if (format.theme === ArticleSpecial.Labs) {
+		return <LabsStandfirst item={item} />;
+	}
+	switch (format.design) {
 		case ArticleDesign.LiveBlog:
-			return css(styles(format), liveblogStyles);
+			return <LiveBlogStandfirst item={item} />;
 		case ArticleDesign.DeadBlog:
-			return css(styles(format), liveblogStyles, deadblogStyles(format));
+			return <DeadBlogStandfirst item={item} />;
+
+		case ArticleDesign.Gallery:
+		case ArticleDesign.Audio:
+		case ArticleDesign.Video:
+			return <MediaStandfirst item={item} />;
+
 		case ArticleDesign.Review:
 		case ArticleDesign.Feature:
 		case ArticleDesign.Editorial:
 		case ArticleDesign.Letter:
 		case ArticleDesign.Comment:
-			return css(styles(format), thinHeadline);
-		case ArticleDesign.Gallery:
-		case ArticleDesign.Audio:
-		case ArticleDesign.Video:
-			return media(format);
-
+			return (
+				<DefaultStandfirst
+					item={item}
+					css={css(
+						defaultStyles(format),
+						css`
+							${headline.xxsmall({ fontWeight: 'light' })}
+						`,
+					)}
+				/>
+			);
 		default:
-			return css(styles(format), normalHeadline);
+			return (
+				<DefaultStandfirst
+					item={item}
+					css={css(
+						defaultStyles(format),
+						css`
+							${headline.xxxsmall({ fontWeight: 'bold' })}
+							padding: 0;
+						`,
+					)}
+				/>
+			);
 	}
```

Comment:
Should these be their own components rather than existing in the `index.ts` file?

Reply:
Yes, as long as we're happy that 4 different layouts use a component that is named for one of them (ie ReviewStandfirst) seeing as the idea was to name our components after layouts not teeny style variations like `<ThinStandfirst />`

Reply:
I think it's fine, although I also think `ThinStandfirst` is reasonable - see the `Metadata` component for examples similar to this. There are always going to be components that look the same across multiple `Designs`/`Displays`, because many of these variations only change some features of the page (e.g. the headline) but not others 🤷.